CalibTracker/SiStripCommon/src/SiStripDCSStatus.cc

```diff
@@ -97,13 +95,10 @@ bool SiStripDCSStatus::getStatus(edm::Event const& e, edm::EventSetup const& eSe
 //
 void SiStripDCSStatus::initialise(edm::Event const& e, edm::EventSetup const& eSetup) {
   //Retrieve tracker topology from geometry
-  edm::ESHandle<TrackerTopology> tTopoHandle;
-  eSetup.get<TrackerTopologyRcd>().get(tTopoHandle);
-  const TrackerTopology* const tTopo = tTopoHandle.product();
+  const auto& tTopo = eSetup.getData(tTopoToken_);
+  const auto& fedCabling_ = eSetup.getData(fedCablingToken_);
```

Comment:
Pardon my ignorance, but given that `initialise()` is run once per job (and per EDM stream, or at least until an event with a valid `FEDRawDataCollection` has been encountered), is `SiStripFedCablingRcd` guaranteed to have a long-enough IOV that it won't get an update within a single job?

Reply:
The FED cabling is changed maybe once / twice per year (if necessary).